department-of-veterans-affairs/va.gov-cms
# VACMS-6885 Claro admin theme proof of concept

## Description

Relates to #6885. Creates a new admin theme, a subtheme of Claro. Extremely a stub theme at this point, no build process scripts or gulp watching or anything like that included yet... test.css styles would be broken into appropriate scss files when that is included.

I did copy over some of the custom js library, ckeditor stylesheet, and .theme code that exists in the current vagovadmin theme.

Overrides the out-of-box Claro alert template with one that more closely matches the alerts seen here: https://design.va.gov/components/alertboxes

Also takes a first crack at updating the default login screen for the CMS to something that feels "more official".

## Testing done
1. install claro, install vagovclaro subtheme
2. verify existing admin theme (vagovadmin) works
3. set vagovclaro as admin theme
4. set vagovclaro as default theme (for login page)

## Screenshots
login screen update:
<img width="1065" alt="Screen Shot 2021-11-10 at 12 23 40 PM" src="https://user-images.githubusercontent.com/11279744/141520831-0aa40f5f-7ae6-4120-8165-086504e81f67.png">

updated claro alert to align with VADS alerts:
<img width="1786" alt="Screen Shot 2021-11-10 at 4 03 59 PM" src="https://user-images.githubusercontent.com/11279744/141520857-ddd314f5-ca42-46c2-bdc6-50f3e0e32d65.png">

## QA steps
As user with admin role
1. Visit appearance page
2. install Claro, set as admin theme. Notice the alert on the page is very garish.
3. install vagovclaro, set as admin theme & default theme. Admin theme handles admin pages, default theme will handle the login page.
3a. See the alert looks nice now.
4. logout, see the new login screen

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?

No announcement needed as the theme will not be enabled, existing users should have no idea of any changes, unless they visit the "appearance" admin page and see the new theme available as an option there.
